ops-devnet/docker-compose.yml

```diff
@@ -107,7 +107,7 @@ services:
 
   l2:
     pid: host # allow debugging
-    image: kromanetwork/geth:v0.5.0
+    image: kromanetwork/geth:v0.5.4
```

Comment:
Rebase after [#391 merged](https://github.com/kroma-network/go-ethereum/pull/122) merged

Reply:
I think it's better to just revert this changes in this PR and bump up geth version in #390